src/backend/sgx/attestation.rs

```diff
@@ -4,27 +4,37 @@
 // * https://github.com/fortanix/rust-sgx for examples of AESM requests.
 
 use super::AESM_SOCKET;
+use crate::backend::sgx::sgx_cache_dir;
 
 use crate::protobuf::aesm_proto::{
     Request, Request_GetQuoteExRequest, Request_GetQuoteSizeExRequest,
     Request_GetSupportedAttKeyIDNumRequest, Request_GetSupportedAttKeyIDsRequest,
     Request_InitQuoteExRequest, Response,
 };
 
+use anyhow::Context;
+use der::{Document, Encode, Sequence};
+use protobuf::Message;
+use sallyport::item::enarxcall::sgx::TargetInfo;
```

Comment:
This is not correctly placed

Reply:
Resolved according to how the imports were sorted in `enarxcall.rs`